Source/OCMockTests/OCMockObjectProtocolMocksTests.m

```diff
@@ -16,19 +16,13 @@
 
 #import <XCTest/XCTest.h>
 #import <OCMock/OCMock.h>
+#import "TestProtocol.h"
```

Comment:
Please do not move classes that are for tests only to their own files. This just increases clutter.